book/website/reproducible-research/reviewing/reviewing-recommend.md

```diff
@@ -5,23 +5,28 @@
 
 Within small-scale projects where the developers all typically already know each other,
 common practice is for the coder to tag someone in the group as the reviewer.
-When you are tagged as a reviewer, first check if you have enough knowledge about
-the new changes in the code to do a good job in a reasonable amount of time.
 
 In contrast, large-scale development projects will likely have existing, concrete rules for
 how reviewers are allocated to individual pull requests.
 These rules serve to balance the group workload and to maximise the various benefits of the process to the project and its participants.
 The very largest projects may even have dedicated staff - or teams of staff - to act as reviewers.
 Typically, code reviews can only be performed by an authorised subset of contributors within larger projects.
 
+If you are a contributor to a more loosely managed or distributed project, a good rule of thumb is to aim to perform reviews in similar quantities to the material you generate for review.
+
 For projects where multiple rounds of review on similar material are likely and long development cycles are anticipated, a degree of strategic thinking on who completes reviews is sensible.
 A single reviewer is likely to be able to make comments on code they have reviewed before much more efficiently.
 However, letting reviewer-coder pairs like this persist is generally a bad idea, as it can lead to the same kinds of groupthink that the review process is designed to avoid in the first place.
 
+Open source projects often also have their own dynamics.
+Contributions may be split between a set of core maintainers, and a larger set of more dispersed contributors, some of whom may have very limited interactions with a project.
+In open source projects it's typically the case that anyone can review a pull request, just as anyone can create one.
+Nevertheless core maintainers, or those who have merge privileges, inevitably have the final say over what gets added to the code, and so also have the largest role when it comes to code review.
```

Comment:
```suggestion
Those who have merge privileges, often limited to core maintainers, inevitably have the final say over what gets added to the code, and so also have the largest role when it comes to code review.
```

I would also add something here about the fact that contribution guidelines should make it explicit who has the final word and why

Reply:
Thanks, noted, will do!